lang/en_AU/texts/nutriscore.html

```diff
@@ -4,10 +4,10 @@ <h2>What is Nutri-Score?</h2>
 
 <div class="row">
 <div class="small-6 medium-6 large-8 xlarge-10 columns">
-<img src="https://static.openfoodfacts.org/images/misc/nutriscore-a.png" alt="NutriScore">
+<img src="https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg" alt="NutriScore">
 <p>The Nutri-Score is a logo that shows the nutritional quality of food products with A to E grades. With the NutriScore, products can be easily and quickly compared.</p>
 Open Food Facts has started computing Nutri-Score in 2014 on its pioneer mobile app and website, while it was only a research paper, and has accompanied its growth from the "5 color score" to the "Nutri-Score" you see in supermarkets today.
-For more on the history of Open Food Facts and the Nutri-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>. Public Health France has supported OpenFoodFacts since 2018 to facilitate consumer information. For more on the history of Open Food Facts and the Nutri-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>.
+Public Health France has supported OpenFoodFacts since 2018 to facilitate consumer information. For more on the history of Open Food Facts and the Nutri-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>. However, the Nutri-Score is not affixed to all products and OpenFoodFacts allows you to have this information by scanning the product.
```

Comment:
```suggestion
However, the Nutri-Score is not affixed to all products and OpenFoodFacts allows you to have this information by scanning the product. Public Health France has supported OpenFoodFacts since 2018 to facilitate consumer information. For more on the history of Open Food Facts and the Nutri-Score, <a href="https://world.openfoodfacts.org/open-food-facts-and-the-nutriscore">you can read our dedicated page</a>. 
```